project_generator/util.py

```diff
@@ -63,13 +64,17 @@ def merge_recursive(*args):
         return reduce(operator.add, args)
 
 def flatten(S):
+    if not isinstance(S, list):
+        return S
```

Comment:
how does this condition differ from the one below? This tests if S is not a list, the one below tests if S is a list?